adoc-mode.el

```diff
@@ -3493,7 +3757,11 @@ Turning on Adoc mode runs the normal hook `adoc-mode-hook'."
                    2 3 nil (1 . nil))))
   (when (boundp 'compilation-error-regexp-alist)
     (make-local-variable 'compilation-error-regexp-alist)
-    (add-to-list 'compilation-error-regexp-alist 'asciidoc)))
+    (add-to-list 'compilation-error-regexp-alist 'asciidoc))
+  (when adoc-show-images-at-startup
```

Comment:
I'd probably name this `adoc-display-images` for consistency with the rest of the naming. After all the fact that the the var is used on mode start is an implementation detail. Technically speaking you can also use it for the toggle code and it can make it simpler.

Reply:
For consistency I changed it to `adoc-display-images-at-startup`. But this variable cannot be used for toggling. It is really only an option whether `adoc-display-images` should be called within `adoc-mode` or not.

The user has the possibility to create or remove images locally through the functions `adoc-display-image-at` und `adoc-remove-image-at`.